### ISSUE TITLE
[FR] Support Amazon Bedrock models

### ISSUE THREAD

**Issue by svaeyens** [logancyang/obsidian-copilot#452]
It would be nice to have the Amazon (AWS) Bedrock foundation models supported. I'm thinking about Claude, Mistral, Llama,...
I know Claude is already supported via Anthropic, but it would be very usefull for me to access the model via Bedrock.

**Comment by roijalbaker** [comment]
I completely agree, it would be much easier to handle a single API key instead of many. Plus you'd only have to set up billing once to access many different models.

**Comment by logancyang** [comment]
@roijalbaker do Amazon Bedrock models work with "openai-format" in custom model?

**Comment by amrtaher1234** [comment]
+1 to this, bedrock is such a common tool nowadays that it should be supported out of the box.

**Comment by isaaclen-lego** [comment]
@logancyang Yes, there are OpenAPI format abstractions for Bedrock, such as https://github.com/aws-samples/bedrock-access-gateway/ among others, so +1
